### PR TITLE
Add author of gist and original author (forked gist)

### PR DESCRIPTION
**Pull request by romgere**
https://github.com/ember-cli/ember-twiddle/issues/403 Add a footer with author informations + original author and link to open gist in ember-twiddle for forked gist.
127 0 0 1_4200_0bceff193ce347a84109b14754f51f71

### CODE REVIEW

File: app/templates/components/user-profil-link.hbs [hunk 1]
```diff
@@ -0,0 +1 @@
+<img src={{user.avatarUrl16}} />{{user.login}}
```
Comment (knownasilya) [inline app/templates/components/user-profil-link.hbs]: This component should be named `user-profile-link`

File: app/components/gist-footer.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+  tagName : 'footer',
```
Comment (knownasilya) [inline app/components/gist-footer.js]:
```suggestion
  tagName: 'footer',
```

File: app/styles/_footer.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    position: relative;
+    z-index: 100;
+    opacity: 1;
+    text-align: center;
+  }
+  color: #666;
+  background: $footer-bg;
+  padding: 0 1.5rem 0;
+
+  a{
```
Comment (knownasilya) [inline app/styles/_footer.scss]:
```suggestion
  a {
```

File: app/styles/_footer.scss [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    color: #666;
+    cursor: pointer;
+
+    &:hover{
```
Comment (knownasilya) [inline app/styles/_footer.scss]:
```suggestion
    &:hover {
```

File: app/styles/_footer.scss [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    img {
+      width: 16px;
+      height: 16px;
+      border-radius: 3px;
+      margin-right: 2px;
+      margin-top: -2px;
+    }
+  }
+
+  .gist-link{
```
Comment (knownasilya) [inline app/styles/_footer.scss]:
```suggestion
  .gist-link {
```

File: app/styles/_twiddle-panes.scss [hunk 6]
```diff
@@ -10,6 +10,10 @@
   @media (min-width: $screen-md-min) {
     display: -webkit-flex;
     display: flex;
+    bottom: $footer-height;
+  }
+  @media (max-width: $screen-md-min) {
+    padding-top: $footer-height
```
Comment (knownasilya) [inline app/styles/_twiddle-panes.scss]:
```suggestion
    padding-top: $footer-height;
```

File: app/templates/components/gist-footer.hbs [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+{{else}}
+  No author (new twiddle)
+{{/if}}
+
+{{#if originalGist}}
+  |
+  Fork from: {{user-profil-link user=originalGist.owner}}'s gist
+  |
+  {{#link-to 'gist.edit' originalGist.id class="gist-link" title="Open original gist in a new ember-twiddle" target="_blank"}}
+    <span class="glyphicon glyphicon-new-window" ></span>&nbsp;Open original gist
```
Comment (knownasilya) [inline app/templates/components/gist-footer.hbs]:
```suggestion
    <span class="glyphicon glyphicon-new-window"></span>&nbsp;Open original gist
```

File: app/templates/components/main-gist.hbs [hunk 8]
```diff
@@ -44,3 +44,5 @@
             exitFullScreen=(action "exitFullScreen")
             switchTests=(action "switchTests")
 }}
+
+{{gist-footer owner=model.owner originalGist=model.forkOf }}
```
Comment (knownasilya) [inline app/templates/components/main-gist.hbs]:
```suggestion
{{gist-footer owner=model.owner originalGist=model.forkOf}}
```

File: app/templates/components/user-profil-link.hbs (hunk 1 above)
Comment (romgere) [reply app/templates/components/user-profil-link.hbs]: Done !

**Pull request merged**